build/docker/mysql.conf.d/custom.cnf

```diff
@@ -15,5 +15,5 @@ max_connections = 900
 character-set-server = utf8
 sql_mode = ""
 innodb = FORCE
-default-storage-engine = InnoDB
```

Comment:
Not sure if we really okay with this. I'll check whether if this is a good practice though 🤔

Reply:
The default storage engine is already InnoDB. I am not sure if this makes any difference. But no harm in keeping it.